ext/standard/string.c

```diff
@@ -6139,3 +6139,133 @@ PHP_FUNCTION(utf8_decode)
 	RETURN_STR(php_utf8_decode(arg, arg_len));
 }
 /* }}} */
+
+/* {{{ proto string unaccent(string str)
+   Removes acents (diacritic sings) from string */
+PHP_FUNCTION(unaccent) {
+
+    zend_string *string;
+    ZEND_PARSE_PARAMETERS_START(1, 1)
+        Z_PARAM_STR(string)
+    ZEND_PARSE_PARAMETERS_END();
+
+    setlocale(LC_CTYPE,"en_US.UTF-8");
```

Comment:
This approach here will not work, because it forces a new locale by calling the function which is problematic for applications that calls the php userland function `setlocale()`. It is neither thread safe either, but the PHP API does provide ways to deal with that.